### ISSUE TITLE
Crash with provided LIBEAY32.dll

### ISSUE THREAD

**Issue by juanmalunatic** [FelixdelasPozas/TrayWeather#115]
**The problem**
Fresh install under Windows 10. App refuses to open due to LIBEAY.dll 1.0.2.8. (No error messages, just silent failure).

**The fix**
I replaced the "faulty" dll for another version that was laying around on existing programs in my PC, and the app now launches.

Here are the relevant lines from Windows' Error Reporting:
```
Sig[0].Name=Application Name
Sig[0].Value=TrayWeather.exe
Sig[1].Name=Application Version
Sig[1].Value=1.21.2.0
Sig[2].Name=Application Timestamp
Sig[2].Value=62a1926c
Sig[3].Name=Fault Module Name
Sig[3].Value=LIBEAY32.dll
Sig[4].Name=Fault Module Version
Sig[4].Value=1.0.2.8
Sig[5].Name=Fault Module Timestamp
Sig[5].Value=579330f8
Sig[6].Name=Exception Code
Sig[6].Value=c0000005
Sig[7].Name=Exception Offset
Sig[7].Value=0000000000012823
```

**Comment by adem4ik** [comment]
Can't reproduce on my Windows 10 even with fresh install:
![image](https://user-images.githubusercontent.com/4707112/175972932-a67707ee-ecc8-49b8-95fc-7dfe29957268.png)

Must be something on your side, like corrupted download/installation.

Anyway, here is dll from my installation, try to drop it and check once more:
[libeay32.zip](https://github.com/FelixdelasPozas/TrayWeather/files/8993153/libeay32.zip)

**Comment by FelixdelasPozas** [comment]
Hi @juanmalunatic , 
I've updated the version of both libeay32 and ssleay32 to 1.0.2.21 in the [latest release (1.22)](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.22.0). Give it a try and please report if it works now.
I have the application in my work machine (Win 10) without problems. Maybe a third party is blocking the dll? An antivirus or firewall? Both ssleay and libeay are from [OpenSSH](https://www.openssh.com/) so older versions could have important bugs and could be banned as a precaution. Or maybe, as @adem4ik says, is just file corruption. 

Regards, 
Félix.

**Comment by juanmalunatic** [comment]
Hey guys, thanks for your prompt response. I've tried the latest release and it works flawlessly: updating the library has apparently fixed the issue.

I was curious so I found [a related BOINC bug](https://github.com/BOINC/boinc/issues/2470): libeay32.dll was causing segfaults  for some newer Intel CPUs (circa 2018). Maybe that's what happened here and why it was hard to reproduce? Anyways, thanks for your time @adem4ik and @FelixdelasPozas :)
